platform/default/src/mbgl/storage/offline_database.cpp

```diff
@@ -1275,7 +1275,8 @@ bool OfflineDatabase::evict(uint64_t neededFreeSize, DatabaseSizeChangeStats& st
         const uint64_t tileChanges = tileQuery.changes();
 
         // Update current ambient cache size, based on how many bytes were released.
-        newAmbientCacheSize = std::max<int64_t>(newAmbientCacheSize - stats.bytesReleased(), 0u);
+        newAmbientCacheSize = std::max<int64_t>(
+            static_cast<int64_t>(newAmbientCacheSize) - static_cast<int64_t>(stats.bytesReleased()), 0u);
```

Comment:
optional nit: a shorter syntax `int64_t(newAmbientCacheSize)`

Reply:
hmmm, I am a little bit hesitated of doing the change, as https://en.cppreference.com/w/cpp/language/explicit_cast explains functional cast is exactly equivalent to the corresponding C-style cast expression.